index.html

```diff
@@ -50,5 +50,196 @@ <h1>Hey, I'm Aidan</h1>
+                    </div>
+                </div>
+             </div>
+             <div class="work">
+                 <div class="multi-post">
+                     <div class="bottom">
+                          <p>Multi-Post Stories</p>
+                          <p>Gain+Glory</p>
+                     </div>
+                     <div class="language">
+                         <ul class="lang">
+                             <li class="lang-list"><button>Ruby on rails</button></li>
+                             <li class="lang-list"><button>css</button></li>
+                             <li class="lang-list"><button>JavaScript</button></li>
+                             <li class="lang-list"><button>html</button></li>
+                         </ul>
+                     </div>
+                     <div class="see">
+                         <button>See Project</button>
+                     </div>
+                 </div>
+              </div>
+              <div class="work">
+                 <div class="multi-post">
+                     <div class="bottom">
+                          <p>Multi-Post Stories</p>
+                          <p>Gain+Glory</p>
+                     </div>
+                     <div class="language">
+                         <ul class="lang">
+                             <li class="lang-list"><button>Ruby on rails</button></li>
+                             <li class="lang-list"><button>css</button></li>
+                             <li class="lang-list"><button>JavaScript</button></li>
+                             <li class="lang-list"><button>html</button></li>
+                         </ul>
+                     </div>
+                     <div class="see">
+                         <button>See Project</button>
+                     </div>
+                 </div>
+              </div>
+        </div>
+    </section>
+    <section>
+        <div class="aboutme">
+            <h1 class="recent">
+                About me
+            </h1>
+            <p class="aboutme-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <div class="resume">
+                <button>Get My Resume</button>
+            </div>
+        </div>
+     </section>
+     <section>   
+        <div id="card">
+            <div class="card-language">
+                <div class="icon">
+                    <img src="./images/Icons.png" alt="icon">
+                </div>
+                <div class="text">
+                    <p>Languages</p>
+                </div>
+                <div class="language-list">
+                    <button>JavaScript</button>
+                    <button>Ruby</button>
+                    <button>Html</button>
+                    <button>CSS</button>
+                </div>
```

Comment:
- [ ] Please replace the `button` here with an `li` and the `div` with an `ul` tag since these are supposed to be a list of languages you have mastered. Additionally, the languages are not supposed to be clickable so putting them in a button might confuse screen readers and ruin the semantic integrity of the web app. Kindly do these for frameworks and skills aswell

Reply:
Resolved